src/ui/NoNameFacet/NoNameFacet.ts

```diff
@@ -0,0 +1,49 @@
+import { Component } from '../Base/Component';
+import { IComponentBindings } from '../Base/ComponentBindings';
+import { ComponentOptions } from '../Base/ComponentOptions';
+import { Initialization } from '../Base/Initialization';
+import { ResponsiveFacets } from '../ResponsiveComponents/ResponsiveFacets';
+import { ResponsiveFacetOptions } from '../ResponsiveComponents/ResponsiveFacetOptions';
+import { exportGlobally } from '../../GlobalExports';
+import { NoNameFacetHeader } from './NoNameFacetHeader';
+import { NoNameFacetOptions, INoNameFacetOptions } from './NoNameFacetOptions';
+
+export class NoNameFacet extends Component {
+  static ID = 'NoNameFacet';
+  static doExport = () => exportGlobally({ NoNameFacet });
+  static options: INoNameFacetOptions = { ...NoNameFacetOptions, ...ResponsiveFacetOptions };
+
+  private header: NoNameFacetHeader;
+
+  constructor(
+    public element: HTMLElement,
+    public options: INoNameFacetOptions,
+    bindings?: IComponentBindings,
+    noNameFacetClassId: string = NoNameFacet.ID
+  ) {
+    super(element, noNameFacetClassId, bindings);
+    this.options = ComponentOptions.initComponentOptions(element, NoNameFacet, options);
+
+    ResponsiveFacets.init(this.root, this, this.options);
+
+    this.ensureDom();
+  }
+
+  public createDom() {
+    this.createContent();
+  }
+
+  private createContent() {
```

Comment:
There will be more stuff inside createDom in the future
There will also be more content than the header, it seems over divided for now but in the near future it will be more complete and meaninful